.env.example

```
@@ -158,6 +158,11 @@ CONNECTOR_OAUTH_CALLBACK_BASE_URL=
 # When enabled, chunk content is also saved to knowledge_documents.chunks column
 # BACKEND_CHUNK_STORAGE_ENABLED=False
 
+# Sensitive-data encryption. Keep these values stable after deployment.
+# Generate exactly 32 and 16 characters as documented.
+GIT_TOKEN_AES_KEY=
+GIT_TOKEN_AES_IV=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GIT_TOKEN_AES_IV key should go before the GIT_TOKEN_AES_KEY key

(UnorderedKey)